jbehave-support-core/src/test/groovy/org/jbehavesupport/core/web/StaleElementExceptionTest.groovy

```diff
@@ -0,0 +1,72 @@
+package org.jbehavesupport.core.web
+
+import org.jbehave.core.model.ExamplesTable
+import org.jbehavesupport.core.TestConfig
+import org.jbehavesupport.core.TestContext
+import org.jbehavesupport.core.expression.ExpressionEvaluatingParameter
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.test.context.ContextConfiguration
+import spock.lang.Specification
+
+@ContextConfiguration(classes = TestConfig.class)
```

Comment:
Shouldnt TestConfig or this test have `@EnableRetry`?

Reply:
Default config is enought